bundle/edu.gemini.spdb.reports.collection/src/main/scala/edu/gemini/dbTools/ephemeris/TryExport.scala

```diff
@@ -0,0 +1,27 @@
+package edu.gemini.dbTools.ephemeris
+
+import scalaz._, Scalaz._
+import scalaz.effect.IO
+
+/** Convenience methods for working with TryExport. */
+object TryExport {
+  def liftExport[A](a: IO[ExportError \/ A]): TryExport[A] =
```

Comment:
I would probably just inline this object in the package object under the type alias.